.github/workflows/test.yml

```diff
@@ -32,7 +32,7 @@ jobs:
           go-version-file: go.mod
           check-latest: true
 
-      - uses: golangci/golangci-lint-action@v5
+      - uses: golangci/golangci-lint-action@v6
```

Comment:
Interesting, the comment says "from 4 to 6" but actually its "from 5 to 6".

Reply:
there was already PR update from 4 to 5.
After updating this branch with the earlier charges the confusion popped w.r.t the name of the PR